src/ui/ClinicalPanel/index.tsx

```diff
@@ -0,0 +1,300 @@
+            value: row.label
+        };
+
+        if (row.hrDetect) {
+            hrDetectData.push(formattedData);
+        } else {
+            otherData.push(formattedData);
+        }
+    });
+
+    return (
+        <div className={`signatures panel-section ${isExpanded ? 'expanded' : 'collapsed'}`}>
+            <button onClick={() => setIsExpanded(!isExpanded)} className="section-header">
+                <h3>Mutational Signatures</h3>
+                <div className="section-toggle">
+                    <svg className="panel-icon-arrow" viewBox={ICONS.TRIANGLE_DOWN.viewBox}>
+                        <title>TRIANGLE DOWN</title>
+                        <path fill="currentColor" d={ICONS.TRIANGLE_DOWN.path[0]} />
+                    </svg>
+                </div>
+            </button>
+            <div className="callout hrdetect">
+                <span className="label">HRDetect</span>
+                <div className="value">
+                    <svg
+                        className={`circle-icon ${isHrDetect ? 'positive' : 'negative'}`}
+                        viewBox={ICONS.CIRCLE.viewBox}
+                    >
+                        <title>CIRCLE</title>
+                        <path fill="currentColor" d={ICONS.CIRCLE.path[0]} />
+                    </svg>
+                    <span>{isHrDetect ? 'Positive' : 'Negative'}</span>
+                </div>
+            </div>
+            <div tabIndex={isExpanded ? 0 : -1} className="section-body">
+                <ToggleRowGroup
+                    callout={'Click to show in the visualization'}
+                    header={'HRD-related Features Found'}
+                    data={hrDetectData}
+                />
+                <ToggleRowGroup header={'Other Features Found'} data={otherData} />
+            </div>
+        </div>
+    );
+};
+
+type ClinicalPanelProps = {
+    demo: any;
+    gosRef: any;
+    filteredSamples: any;
+    isClinicalPanelOpen: boolean;
+    hasClinicalInfo: boolean;
+    setInteractiveMode: (interactiveMode: boolean) => void;
+    setIsClinicalPanelOpen: (isClinicalPanelOpen: boolean) => void;
+};
+
+export const ClinicalPanel = ({
+    hasClinicalInfo,
+    demo,
+    gosRef,
+    isClinicalPanelOpen,
+    setInteractiveMode,
+    setIsClinicalPanelOpen
+}: ClinicalPanelProps) => {
+    const [clinicalInformation, setClinicalInformation] = useState(null);
+
+    const handleZoomToGene = (gene: string) => {
+        setInteractiveMode(true);
+        setTimeout(() => {
+            document.getElementById('variant-view')?.scrollIntoView({
+                block: 'start',
+                inline: 'nearest',
+                behavior: 'smooth'
+            }),
+                0;
+        });
+
+        gosRef.current.api.zoomToGene(`${demo.id}-mid-ideogram`, `${gene}`, 1000);
+    };
+
+    useEffect(() => {
+        if (hasClinicalInfo && demo?.clinicalInfo) {
+            console.log('demo.clinicalInfo', demo.clinicalInfo);
```

Comment:
I assume this was needed only for the purpose of the implementation.

```suggestion
            console.log('demo.clinicalInfo', demo.clinicalInfo);
```